Simplified/NYPLReportIssueViewController.swift

```diff
@@ -10,8 +10,10 @@ import UIKit
 import HelpStack
 import MessageUI
 
-class NYPLReportIssueViewController: UITableViewController, UITextFieldDelegate, UITextViewDelegate, MFMailComposeViewControllerDelegate {
-  
+class NYPLReportIssueViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, UITextFieldDelegate, UITextViewDelegate, MFMailComposeViewControllerDelegate {
+
+  @IBOutlet var tableView: UITableView!
```

Comment:
Everything looks great! Thought you actually do not need an IBOutlet to the tableview here.